src/sfeQwiicUltrasonic.h

```diff
@@ -1,18 +1,26 @@
+/* SparkFun Ulrasonic Distance Sensor 
+ * 
+ * Product: 
+ *  *  SparkFun Qwiic Ultrasonic Distance Sensor - HC-SR04 (SEN-1XXXX)
+ *  *  https://www.sparkfun.com/1XXXX
+ * 
+ * SPDX-License-Identifier: MIT
+ *
+ * Copyright (c) 2024 SparkFun Electronics
+ */
+
 #pragma once
 
 #include "SparkFun_Toolkit.h"
 #include <stdint.h>
 
 // Available I2C addresses of the Qwiic Ultrasonic
-const uint8_t kQwiicUltrasonicAddresses[] = {0x20, 0x21, 0x22, 0x23, 0x24, 0x25, 0x26, 0x27,
-                                             0x28, 0x29, 0x2A, 0x2B, 0x2C, 0x2D, 0x2E, 0x2F};
-const uint8_t kQwiicUltrasonicNumAddresses = sizeof(kQwiicUltrasonicAddresses) / sizeof(uint8_t);
-const uint8_t kQwiicUltrasonicMinAddress = kQwiicUltrasonicAddresses[0];
-const uint8_t kQwiicUltrasonicMaxAddress = kQwiicUltrasonicAddresses[15];
 const uint8_t kQwiicUltrasonicDefaultAddress = 0x2F;
-
-// Register to trigger a new measuremnt and read the previous one
-const uint8_t kQwiicUltrasonicRegisterTrigger = 0x01;
+const uint8_t kQwiicUltrasonicMinAddress = 0x08;
+const uint8_t kQwiicUltrasonicMaxAddress = 0x7F;
```

Comment:
My only concern here is how this will affect boards with old firmware on them. Is there a firmware version register on the device that can be polled to dynamically determine the valid I2C address range? If not, is that something we can add to the firmware? That might have other benefits as well.

Reply:
I went ahead and re -uploaded the previous function and changed the name on the one I added to `updateAddress`. This of course meant that I had to re-add the previous address list.